src/code/z_kanfont.c

```diff
@@ -33,13 +41,36 @@ void Font_LoadMessageBoxIcon(Font* font, u16 icon) {
  * the font buffer.
  */
 void Font_LoadOrderedFont(Font* font) {
+    s32 size;
     s32 len;
     s32 codePointIndex;
     s32 fontBufIndex;
     u32 offset;
 
-    font->msgOffset = FONT_MESSAGE_OFFSET;
-    len = font->msgLength = FONT_MESSAGE_LENGTH;
+#if OOT_NTSC
+    font->msgOffset = _message_0xFFFC_jpn - (const char*)_jpn_message_data_staticSegmentStart;
+    size = font->msgLength = _message_0xFFFD_jpn - _message_0xFFFC_jpn;
```

Comment:
I really didn't like how the previous `FONT_MESSAGE_OFFSET` and `FONT_MESSAGE_SIZE` defines obscure what's going on. In particular, I think the fact that it uses `jpn` instead of `nes` is an important difference, and should be put alongside all the other NTSC vs PAL differences in this function.

I don't think the `_` prefix is a good argument because the `_nes_font_staticSegmentRomStart` symbols have that prefix too and similar symbols appear everywhere in source code.

Reply:
Personally think the defines are fine in this case. And thats coming from me, I usually don't like macros that hide information

Reply:
Alright, I've restored the macros